src/main/java/it/geosolutions/httpproxy/HTTPProxy.java

```diff
@@ -156,30 +156,36 @@ public HttpClient createHttpClient() {
         if (httpClient != null)
             return httpClient;
 
-        HttpHost httpHost = getHost("http.proxyHost", "http.proxyPort");
-        if (httpHost != null) {
-            HttpRoutePlanner routePlanner = new DefaultProxyRoutePlanner(httpHost) {
-                @Override
-                public HttpRoute determineRoute(
-                        final HttpHost host,
-                        final HttpRequest request,
-                        final HttpContext context) throws HttpException {
-                    String hostname = host.getHostName();
-                    if (isNonProxyHost(hostname)) {
-                        // Return direct route
-                        return new HttpRoute(host);
+        final HttpHost httpHost = getHost("http.proxyHost", "http.proxyPort");
+        LOGGER.debug("http host: " + httpHost);
+        final HttpHost httpsHost = getHost("https.proxyHost", "https.proxyPort");
+        LOGGER.debug("https host: " + httpsHost);
```

Comment:
As I say above the HTTP proxy business should be isolated in a method, the log should (and variables) should be more meaningful. 

Imagine someone reading the log and he seems something like this:
```
https host: outputvm.net25
```
Unless you wrote, this means absolutely zero.

While if we have a log like (DEBUG LEVEL):
```
HTTP proxy host: outputvm.net25
HTTPS proxy host: NOT DEFINED
```

Reply:
updated the logs